app/src/androidTest/java/org/theotech/ceaselessandroid/ApplicationTest.java

```diff
@@ -1,13 +1,13 @@
-package org.theotech.ceaselessandroid;
-
-import android.app.Application;
-import android.test.ApplicationTestCase;
-
-/**
- * <a href="http://d.android.com/tools/testing/testing_android.html">Testing Fundamentals</a>
- */
-public class ApplicationTest extends ApplicationTestCase<Application> {
-    public ApplicationTest() {
-        super(Application.class);
-    }
-}
+//package org.theotech.ceaselessandroid;
+//
```

Comment:
should this be cleaned up if its all commented out?

Reply:
Ok. Removing the dummy test cases.